src/packages/frontend/frame-editors/jupyter-editor/cell-notebook/actions.ts

```diff
@@ -421,8 +422,7 @@ export class NotebookFrameActions {
       undefined,
       false,
     );
-    let md_edit_ids = this.store.get("md_edit_ids");
-    if (md_edit_ids == null) md_edit_ids = Set();
+    const md_edit_ids = this.store.get("md_edit_ids", Set());
```

Comment:
This change you just made is NOT equivalent and it pretty dangerous.   If 'md_edit_ids' is actually set to undefined or to null in the store, then ` this.store.get("md_edit_ids", Set());` would be undefined or null, NOT Set().  It's much, much safer to do:

```
const md_edit_ids = this.store.get("md_edit_ids") ?? Set()
```

It's also more efficient since above doesn't evaluate Set() unless needed, whereas ` this.store.get("md_edit_ids", Set());` always does.